packages/react-components/src/components/tests/MessageThread.test.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
test file should be inline beside component instead of its own tests directory

Reply:
(can check with the team what everyone prefers in case the directory gets really long. I prefer beside the src file as then tests are treated as a primary concern, always easy to find the file, and easy to see when a file has no tests (as there is no .test file beside it)

Reply:
+1 to side-by-side.

I've seen no benefit from splitting them. They're supposed to be tightly coupled with the components, so might as well keep together.